index.js

```diff
@@ -79,7 +80,8 @@ const execCompose = (command, args, options) => new Promise((resolve, reject) =>
     result.err += chunk.toString();
   });
 
-  childProc.on('close', () => {
+  childProc.on('close', (exitCode, _) => {
```

Comment:
Actually, `exit` event should be used instead. `close` event is emitted when stdio streams for a process are closed, which might not be the same as process ending, especially when multiple processes are spawned and share stdio.

See:
https://nodejs.org/api/child_process.html#child_process_event_exit
> When the 'exit' event is triggered, child process stdio streams might still be open.

vs
https://nodejs.org/api/child_process.html#child_process_event_close
> The 'close' event is emitted when the stdio streams of a child process have been closed. This is distinct from the 'exit' event, since multiple processes might share the same stdio streams.